assets/src/design-system/components/snackbar/snackbarContainer.js

```diff
@@ -122,30 +124,44 @@ export const SnackbarContainer = ({
   return (
     <StyledContainer placement={placement}>
       <TransitionGroup>
-        {orderedNotifications.map((notification) => (
-          <CSSTransition
-            in
-            key={notification.key}
-            timeout={300}
-            unmountOnExit
-            classNames="react-snackbar-alert__snackbar-container"
-          >
-            <ChildContainer placement={placement}>
-              <Component
-                aria-label={notification.message}
-                placement={placement}
-                onDismiss={handleDismiss(notification)}
-                onAction={notification.onAction}
-                actionLabel={notification.actionLabel}
-                message={notification.message}
-                showCloseButton={Boolean(notification.dismissable)}
-                removeMessageTimeInterval={notification.timeout}
-                isPreventAutoDismiss={notification.preventAutoDismiss}
-                isPreventActionDismiss={notification.preventActionDismiss}
-              />
-            </ChildContainer>
-          </CSSTransition>
-        ))}
+        {orderedNotifications.map((notification) => {
+          const {
+            actionLabel,
+            dismissable,
+            message,
+            onAction,
+            preventActionDismiss,
+            preventAutoDismiss,
+            timeout,
+            ...notificationProps
+          } = notification;
+
+          return (
+            <CSSTransition
+              in
+              key={notification.key}
+              timeout={300}
+              unmountOnExit
+              classNames="react-snackbar-alert__snackbar-container"
+            >
+              <ChildContainer placement={placement}>
+                <Component
+                  {...notificationProps}
```

Comment:
spread the rest of the notification props into the component so that it can have access to `customZIndex`